Header header
string parent_frame_id
sensor_msgs/Image image
rasberry_perception/ObjectAnnotation[] annotations
string[] class_labels
